06 - poslowie/poslowie.js

```diff
@@ -0,0 +1,28 @@
+const poslowieApi = new URL("https://api-v3.mojepanstwo.pl/dane/poslowie/");
+const okreg = "conditions[poslowie.sejm_okreg_id]";
+const krakow = 13;
+
+class Rebels{
+    constructor(api, conditions, param) {
+        this.api = api;
+        this.conditions = conditions;
+        this.param = param;
+    }
+    
+   	displayRebels(data) {
+        this.poslowie = data.map(val => (val.data["poslowie.zbuntowanie"] + "," + val.data["poslowie.nazwa"])).sort().map(val => val.split(","));
```

Comment:
hm, I'm not sure why are doing this like that - adding a comma just to remove it later on?
Also, why do you need a variable here? `this.poslowie` seems unnecessary.

How about doing it like this:

```
data
  .sort(/* pass a function that sorts by name */)
  .forEach(row => console.log(row['poslowie.nazwa'] + " wskaźnik zbuntowania: " + row['poslowie.zbuntowanie])
```